.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Hey Gav, I use .editorconfig in most projects especially when other people are working on them - by default my editor will tab 2 spaces but with this file it will know the project is set to tab 4 spaces - linting will indeed catch any errors, but this stops them from ever being made in the first place. Albeit the first line you have noted can probably be dropped